Marlin/pins_SANGUINOLOLU_11.h

```diff
@@ -240,7 +240,7 @@
     #ifndef ST7920_DELAY_1
       #define ST7920_DELAY_1 DELAY_0_NOP
     #endif
-    #ifndef ST7920_DELAY_3
+    #ifndef ST7920_DELAY_2
```

Comment:
Why are you lowering this delay in the generic Sanguinololu master pins and not the MELZI pins file ?  Also, was this even intentional ?  The test seems illogical as it tests for !2 and then sets to a value that is in fact !2